### ISSUE TITLE
Service account import issues

### ISSUE THREAD

**Issue by Frassle** [pulumi/pulumi-gcp#1375]
### What happened?

Part user issue from slack, part just my observations.

User was trying to import a service account with: `pulumi import gcp:serviceaccount/account:Account default projects/****/serviceAccounts/**@**.iam.gserviceaccount.com` but got an error:
```
gcp:serviceaccount:Account (default):
    error: Preview failed: Resource type 'gcp:serviceaccount/account:Account' not found.
```

When I went to look at the docs for this I noticed the import docs where a mess, including broken markdown from Terraform docs about import blocks.

### Example

`pulumi import gcp:serviceaccount/account:Account default projects/****/serviceAccounts/**@**.iam.gserviceaccount.com`

### Output of `pulumi about`

N/A

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by iwahbe** [comment]
For posterity, the import docs on `gcp:serviceaccount:Account` look like this:

<img width="735" alt="Screenshot 2023-11-28 at 11 43 08 AM" src="https://github.com/pulumi/pulumi-gcp/assets/22222529/7a92b557-94d3-4281-912f-b710c48b68ca">

**Comment by russ-davey** [comment]
I'm also experiencing this problem and it's blocking our IaC pipeline as we can't deploy any changes!

I'm using this package, but SDK v6 also has the same issue:

github.com/pulumi/pulumi-gcp/sdk/v7/go/gcp/serviceaccount

Error message is:

Diagnostics:
  gcp:serviceAccount:Account (log-writer):
    error: Resource type 'gcp:serviceAccount/account:Account' not found.

**Comment by guineveresaenger** [comment]
This should be fixed as of pulumi-terraform-bridge v3.72.0, which will go out bundled in this week's gcp release.

https://github.com/pulumi/pulumi-terraform-bridge/pull/1629 addresses this specifically.

[Here is a preview of the cleaned up Import block](https://github.com/pulumi/registry/pull/3880) 

Because registry site previews don't persist, here's a screenshot:

<img width="833" alt="Screenshot 2024-01-31 at 1 22 45 PM" src="https://github.com/pulumi/pulumi-gcp/assets/13116240/976fe0f9-cc5b-4ab7-b079-5033eaa8a0f1">

This might be good enough to close this issue although of course it would be a nice refinement to ensure we add some newlines.

**Comment by iwahbe** [comment]
Let's take the opportunity to get the newline insertion working. This is taken from http://registry--origin-pr-3880-8a2862d0.s3-website.us-west-2.amazonaws.com/registry/packages/gcp/api-docs/certificatemanager/certificatemap/#gcp-certificatemanager-certificatemap:

<img width="697" alt="Screenshot 2024-02-01 at 11 27 34 AM" src="https://github.com/pulumi/pulumi-gcp/assets/22222529/632cccc7-8f5f-42a7-8bc5-508df05e5d6e">


There is a pretty big readability improvement going from ☝️ to 👇.

> CertificateMap can be imported using any of these accepted formats
> 
> * `projects/{{project}}/locations/global/certificateMaps/{{name}}` 
> * `{{project}}/{{name}}` 
> * `{{name}}` 
> 
> When using the pulumi import command, CertificateMap can be imported using one of the formats above. For example
